gnucash/gnucash-cli.cpp

```diff
@@ -44,9 +44,6 @@
 
 namespace bl = boost::locale;
 
-/* This static indicates the debugging module that this .o belongs to.  */
-static QofLogModule log_module = GNC_MOD_GUI;
```

Comment:
`__attribute__((unused))`

Reply:
This is C++, so I think [[maybe_unused]] is better

Reply:
Yes it is.